json_schemas/agreement-details.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "additionalProperties": false,
+  "minProperties": 1,
+  "properties": {
+    "signerName": {
```

Comment:
Have spoken with Laurence and looked at the admin app. You are right. In his prototype we want to show the name and the email of the user who uploaded the framework agreement. Will discuss with @pcraig3 if it is worth storing this information or getting it on the fly and come to a decision.